README.md

```diff
@@ -1,16 +1,15 @@
 # Open Heart Protocol
 
-Open Heart Protocol lets an anonymous user sends an emoji reaction to a URL.
+The Open Heart Protocol lets an anonymous user sends an emoji reaction to a URL.
 
 ## How
 
-Set up an endpoint to receive an Open Heart POST request like this one:
+Set up an endpoint to receive an Open Heart `POST` request like this one:
 
 ```
 curl -d '🫠'  -X POST '<url>'
 ```
 
```

Comment:
Does this make sense?

```suggestion

A Open Heart message should contain of a single emoji sequence. However, the emoji sequence may be followed by arbitrary data which the server is expected to ignore. 

This allows HTML `<form>`s to post reactions using the Open Heart protocol through an empty input. In this case, the payload will be `🫠=`.
```

Reply:
Yup, I'll include a form markup example here as well.

Reply:
Using https://api.openheart.fyi for the example now.